sos/report/plugins/clear_containers.py

```diff
@@ -21,8 +21,10 @@ class ClearContainers(Plugin, IndependentPlugin):
     runtime = 'cc-runtime'
     packages = (runtime,)
     services = ('cc-proxy',)
+    limit = None
```

Comment:
I had to search in code what is the purpose of the variable. When we touch it, maybe it makes sense to rename it to e.g. `loglimit`? Or is the current naming sufficient?

Reply:
`loglimit` seems better - renamed.